taipy/core/config/job_config.py

```diff
@@ -37,32 +35,34 @@ class JobConfig(UniqueSection):
     _DEVELOPMENT_MODE = "development"
     _DEFAULT_MODE = _DEVELOPMENT_MODE
     _DEFAULT_MAX_NB_OF_WORKERS = 2
-    _MODES = [_STANDALONE_MODE, _DEVELOPMENT_MODE]
+    _MODES = [_DEVELOPMENT_MODE, _STANDALONE_MODE]
```

Comment:
Any specific reason?

Reply:
No, just want to make the default value as the first value of the list 😅